pipeline/package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
nit: I prefer the longest versions of these, as easier to google and more readable to people who aren't super-familiar with bash

```suggestion
#!/usr/bin/env bash
set -o errexit
set -o nounset
set -o pipefail
```